tests/safeds/data/tabular/containers/_column/test_plot_boxplot.py

```diff
@@ -27,3 +30,12 @@ def test_plot_boxplot_int(monkeypatch: _pytest.monkeypatch) -> None:
     monkeypatch.setattr(plt, "show", lambda: None)
     table = Table.from_dict({"A": [1, 2, 3]})
     table.get_column("A").plot_boxplot()
+
+
+def test_plot_boxplot_legacy() -> None:
```

Comment:
```suggestion
def test_should_match_snapshot() -> None:
```

The tested method is already part of the name of the test file.

Reply:
Same for other tests.